File: Task3HWPPT2PDPinJava/app/src/main/java/com/example/task3hwppt2pdpinjava/activities/MainActivity.java
package com.example.task3hwppt2pdpinjava.activities;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;

import com.example.task3hwppt2pdpinjava.R;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        initViews();
    }
    void initViews(){
        Button btnOpenImage = findViewById(R.id.btnOpenImage);
        btnOpenImage.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                OpenPhotoActivity();
            }
        });
    }
    void OpenPhotoActivity(){
        Intent intent = new Intent(this, PhotoActivity.class);
        startActivity(intent);
    }
}

File: Task3HWSecondPPTwithPDPinKotlin/app/src/main/java/com/example/task3hwsecondpptwithpdpinkotlin/activities/MainActivity.kt
package com.example.task3hwsecondpptwithpdpinkotlin.activities

import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.Button
import com.example.task3hwsecondpptwithpdpinkotlin.R

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        initViews()
    }

    fun initViews(){
        var btnOpenImage = findViewById<Button>(R.id.btnOpenImage)
        btnOpenImage.setOnClickListener {
            openPhotoActivity()

        }
    }
    fun openPhotoActivity(){
        var intent = Intent(this, PhotoActivity::class.java)
        startActivity(intent)
    }
}